vosekast_control/Scale.py

```diff
@@ -251,7 +255,8 @@ def read_value_from_scale(self):
 
     def add_new_value(self, new_value):
 
-        self.timestamp = datetime.now()
+        self.timestamp = time.time() * 1000
```

Comment:
why did you store the timestamp in scale class. Are you reusing that value?

Reply:
I am only reusing the values in the deque, so I changed it

Reply:
i only find three places of that timestamp. in the init function. at this place. and short after that where you add that timestamp to the history. So to store that timestamp in the class makes no sense here.